cmd/buildah/config.go

```diff
@@ -18,42 +18,50 @@ const (
 
 var (
 	configFlags = []cli.Flag{
+		cli.StringSliceFlag{
+			Name:  "annotation, a",
+			Usage: "`annotation` e.g. annotation=value, for the target image",
+		},
+		cli.StringFlag{
+			Name:  "arch",
+			Usage: "`architecture` of the target image",
+		},
 		cli.StringFlag{
 			Name:  "author",
 			Usage: "image author contact `information`",
 		},
 		cli.StringFlag{
+			Name:  "cmd",
+			Usage: "`command` for containers based on image",
```

Comment:
This isn't really clear at all. Suggest "set command to run as entrypoint for containers using this image" or similar

Reply:
Modified to talk about setting default command.